testdata/cheats/Cheats.sol

```diff
@@ -135,4 +135,27 @@ interface Cheats {
     function toString(bool)           external returns(string memory);
     function toString(uint256)        external returns(string memory);
     function toString(int256)         external returns(string memory);
+    // Snapshot the current state of the evm.
+    // Returns the id of the snapshot that was created.
+    // To revert a snapshot use `revertTo`
+    function snapshot() external returns(uint256);
+    // Revert the state of the evm to a previous snapshot
+    // takes the snapshot id to revert to. This deletes the snapshot and all snapshots taken after the given snapshot id.
+    function revertTo(uint256) external returns(bool);
+    // Creates a new fork with the given endpoint and block and returns the identifier of the fork
+    function createFork(string calldata,uint256) external returns(uint256);
+    // Creates a new fork with the given endpoint and the latest block and returns the identifier of the fork
+    function createFork(string calldata) external returns(uint256);
+    // takes a fork identifier created by `createFork` and changes the state
+    function selectFork(uint256) external;
```

Comment:
Another question here actually: if you pass a fork URL using the CLI, then do `createFork` etc., how would you revert back to the initial fork passed via CLI? Is it possible? If not, should we discourage passing `--rpc-url`?

Reply:
if we launch in fork mode (via CLI) this fork will always have id `0`, I added another cheatcode `activeFork()(uint256)` that returns the currently active fork, so the id of the CLI fork can be retrieved as well

Reply:
Great! 😄